.github/workflows/slow-checks.yml

```diff
@@ -59,7 +59,13 @@ jobs:
           - int-linux-x64-4core-gpu-t4-ubuntu24.04-1
           - int-linux-x64-4core-ubuntu24.04-1
           - int-linux-x64-2core-ubuntu24.04-1
-    name: Install Pipx on ${{ matrix.runner }}
+        python-version:
+          - "3.9"
```

Comment:
```suggestion
```

3.9 goes EOL in 2 months. Do we really want to have it here

Reply:
Removed it